### PR TITLE
OpenZaak Sync

### PR DESCRIPTION
**Pull request by rjzondervan**

### CODE REVIEW

**Review by bbrands02 — APPROVED** [review]

File: Installation/Source/openzaak.drc.source.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
```
Comment (WilcoLouwerse) [inline Installation/Source/openzaak.drc.source.json:1]: Remove this file, already added this in the predesignated file for this: buren.drc.source.json

File: Installation/Source/openzaak.ztc.source.json [hunk 2]
```diff
@@ -0,0 +1,16 @@
+{
```
Comment (WilcoLouwerse) [inline Installation/Source/openzaak.ztc.source.json:1]: Remove this file, already added this in the predesignated file for this: buren.ztc.source.json

**Review by WilcoLouwerse — CHANGES_REQUESTED** [review]

File: Installation/Action/kiss.SyncZakenAction.action.json [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+      1,
+      1
+    ]
+  },
+  "class": "App\\ActionHandler\\SynchronizationCollectionHandler",
+  "configuration": {
+    "useDataFromCollection": true,
+    "location": "/zaken",
+    "entity": "https://vng.opencatalogi.nl/schemas/zrc.zaak.schema.json",
+    "source": "https://openzaak.nl/source/openzaak.zrc.source.json",
```
Comment (WilcoLouwerse) [inline Installation/Action/kiss.SyncZakenAction.action.json:20]: Use this source instead: buren.zrc.source.json

**Review by WilcoLouwerse — CHANGES_REQUESTED** [review]

File: Installation/Source/buren.drc.source.json [hunk 4]
```diff
@@ -2,5 +2,14 @@
     "title": "DRC API",
     "$id": "https://buren.nl/source/buren.drc.source.json",
     "$schema": "https://docs.commongateway.nl/schemas/Gateway.schema.json",
-    "version": "0.0.1"
+    "version": "0.0.1",
+    "description": "Een API om een documentregistratiecomponent (DRC) te benaderen.",
+    "location": "https://open-zaak.test.buren.opengem.nl/documenten/api/v1",
+    "headers": {
+        "Accept-Crs": "EPSG:4326",
```
Comment (WilcoLouwerse) [inline Installation/Source/buren.drc.source.json:9]: watch out, missing accept header

**Review by WilcoLouwerse — APPROVED** [review]
I already pushed these changes to these sources, as we had discussed i should do, earlier today.
So you don't have to change them, only the Action
(Watch out with the merge conflicts)

**Pull request merged**